pkg/aws/ocpgwdeployer.go

```diff
@@ -43,6 +45,11 @@ type ocpGatewayDeployer struct {
 
 var PreferredInstances = []string{"c5d.large", "m5n.large"}
 
+var (
+	nodeSGSuffix         = "-worker-sg"
+	controlPlaneSGSuffix = "-master-sg"
```

Comment:
These are mutable global vars which is inherently unsafe. Perhaps they should be stored in the `awsCloud` instance?

Reply:
Done.